PRS8001-9000/PRS8788Severos.xml

```diff
@@ -39,21 +39,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
+         <change who="MKr" when="2020-06-06">Added external links</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
+            <person sex="1" sameAs="wd:Q2637658">
                <persName>
-                  
                   Severos of Ašmunayn
                </persName>
-               <persName type="alt">Sawiros zäʾǝsmunayn</persName>
-               <persName type="alt">Sawīros b. al-Muqaffaʿ</persName>
-               <occupation> rel. </occupation>
+               <persName xml:lang="gez" xml:id="n1" type="alt">Sāwiros zäʾǝsmunāyn</persName>
+               <persName xml:lang="gez" corresp="#n1" type="alt">ሳዊሮስ፡ ዘእስሙናይን፡</persName>
```

Comment:
za-ʾƎsmunāyn

Reply:
ał transliterated names should have type normalized and i would not take type alt for the Name in fidel.